utils/check_repo.py

```diff
@@ -83,6 +83,9 @@
     "UMT5EncoderModel",  # Building part of bigger (tested) model.
     "Blip2QFormerModel",  # Building part of bigger (tested) model.
     "ErnieMForInformationExtraction",
+    "FastSpeech2ConformerHifiGan",  # Already tested by SpeechT5HifiGan (# Copied from)
+    "FastSpeech2ConformerWithHifiGan",  # Built with two smaller (tested) models.
+    "GraphormerEncoder",  # Building part of bigger (tested) model.
```

Comment:
This is probably nothing but why does `GraphormerEncoder` appears added in your PR ?

Reply:
Yep it's from a rebase, removed